tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -1863,9 +1868,10 @@ namespace {
 bool MayPreventVectorization(const HloInstruction& hlo) {
   if (hlo.opcode() == HloOpcode::kFusion) {
     return absl::c_any_of(hlo.fused_instructions_computation()->instructions(),
-                          [](const HloInstruction* instr) {
+                          [&](const HloInstruction* instr) {
                             switch (instr->opcode()) {
                               case HloOpcode::kReduce:
```

Comment:
Since `false` is default, just removing it from the list would suffice?

Reply:
Also could you add a test case affected by this particular the change in this particular function? I'm not sure what would be a good testing boundary, maybe LLVM?

Reply:
You are right that I could remove it. But as I want to explicitly to return false, I moved it the case to just before the default case. I see this as having a whitelist and a blacklist and a default. Even if the default is the same as the black list, I prefer to have it explicit. What do you think of the new version?

As the behavior is the same as the default value, I'm not sure how to test this. Any idea?